examples/example.yaml

```diff
@@ -30,13 +30,13 @@ spec:
           requests:
             storage: "60Gi"
         volumeMode: Filesystem
-  resources:
-    requests:
-      cpu: "2"
-      memory: "8Gi"
-    limits:
-      cpu: "2"
-      memory: "8Gi"
+  # resources:
```

Comment:
Clean up when you are done

Reply:
I ended up updating the template for this. Since it's an example (not a production config), I dropped the requests down to 25% of the limits. This should be sufficient for running it in kind, but still allow it to scale up for tests in GKE/EKS, etc.